packages/deployment/app/cloudformation.template.yml

```diff
@@ -591,96 +545,29 @@ Resources:
   # DynamoDB config END
   #################################################
 
+{{# if dynamo2ElasticSearch}}
```

Comment:
The templating for this used by the API template for no reason that I could see, since it really has nothing to do with APIs. So I moved this to be handled by the parent stack